MoviesStoreApp/MoviesStoreMVVM/TableViewCells/MovieInfoTableViewCell.swift

```diff
@@ -121,28 +124,42 @@ final class MovieInfoTableViewCell: UITableViewCell {
 
     // MARK: - Public Methods
 
-    func refreshData(_ model: MovieInfo) {
+    func configure(_ model: MovieInfo, movieInfoViewModel: MovieInfoViewModelProtocol) {
         var genres = ""
         var countries = ""
```

Comment:
```suggestion
        var genres = ""
        var countries = ""
```

Constants.emptyString ??

Reply:
Женя вроде говорил что не выносим больше